src/azure-cli/azure/cli/command_modules/sql/aaz/latest/sql/mi/startstopschedule/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "sql mi startstopschedule",
```

Comment:
```suggestion
    "sql mi start-stop-schedule",
```
Better to split with `-` or name `sql mi schedule` directly

Reply:
Fixed